packages/ui/components/button/Button.tsx

```diff
@@ -171,7 +171,8 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
                 name={StartIcon}
                 className={classNames(
                   variant === "icon" && "h-4 w-4",
-                  variant === "button" && "h-4 w-4 stroke-[1.5px] ltr:-ml-1 ltr:mr-2 rtl:-mr-1 rtl:ml-2"
+                  variant === "button" &&
+                    "h-4 w-4 shrink-0 stroke-[1.5px] ltr:-ml-1 ltr:mr-2 rtl:-mr-1 rtl:ml-2"
```

Comment:
unnecessary changes

Reply:
@anikdhabal : I have just added shrink-0 class. If i don't apply shrink-0 then icon gets squeezed. Attaching Screenshot for reference
![image](https://github.com/user-attachments/assets/ad740406-c5dd-4ad6-824e-aabccb4ede2f)

Reply:
This is used globally. For small bug, I prefer not to update this. Please use a different method

Reply:
@anikdhabal : I do understand it is used at global level, but is there any place where we want to shrink the icon? Even in the same booking page i noticed shrink-0 is applied to icons. 

Also it is difficult to fix this issue without modifying Button.tsx file. Other than adding the class directly i have 2-3 approaches mentioned below. Please let me know which works best or feel free to suggest an alternate approach
1) add a new prop like startIconClasses and include it in Icon classname.
2) pass the icon as customstartIcon (Note: need to copy all the button classes and pass it as icon from Component.tsx)
3) create a new property for shrink in buttonClasses which in turn provides the ability to control shrink based on props.